### ISSUE TITLE
Feature: Support gzip plugin in dashboard

### ISSUE THREAD

**Issue by bingoku** [apache/apisix-dashboard#2088]
### Issue description

通过 ETCD 添加 gzip 插件配置后 apisix 志提示

![image](https://user-images.githubusercontent.com/4134788/130737235-5e7e34a0-0f01-45a1-9863-ffc1a4fb8a72.png)

```
2021/08/25 14:14:00 [error] 50#50: *3001 [lua] gzip.lua:97: phase_func(): need to build APISIX-OpenResty to support setting gzip while reading response header from upstream, client: 10.0.0.3, server: _, request: "GET / HTTP/2.0", upstream: "http://172.16.255.212:3000/",
```
查看文档说需要重新编译
https://raw.githubusercontent.com/api7/apisix-build-tools/master/build-apisix-openresty.sh 重新编译，
根据这个脚本，编译成功以后，运行新的容器
上述日志提示消失
```
curl -H "Accept-Encoding: gzip" -i https://logs.ooxx.cn
```
![image](https://user-images.githubusercontent.com/4134788/130736516-e62321b1-4e53-433b-b633-07c58185a61b.png)

这个是稳定复现的，应该是 apisix-dashboard 的问题。

### Expected behavior

通过 UI  能正常进行设置

### How to Reproduce

重新编译以后，日志提示消失，但是 UI 上依然显示上述日志。

### Screenshots

![image](https://user-images.githubusercontent.com/4134788/130735832-6a88c181-f4aa-4468-b88b-57a7f022d904.png)

### Environment

apisix 2.9
apisix-dashboard 最新 master

**Comment by liuxiran** [comment]
Thanks for your report @bingoku , APISIX-dashboard 2.7.2 can works well only Apache APISIX 2.7. And gzip plugin is added in ver 2.8.0, see here: https://github.com/apache/apisix/blob/master/CHANGELOG.md#280

We 'll fix it in our 2.8 version, and would you like do handle this issue?

Then at the same time, would you please make your example data a little more elegant? It's better that way as a valuable issue.

**Comment by liuxiran** [comment]
Hi @bingoku 

it has already been fixed in our 2.8 version, you can use `Raw Data Editor` to configure the `gzip` plugin.

But  there is another bug here, when configure `gzip` plugin in `Plugin Editor` there is an error happened:

![image](https://user-images.githubusercontent.com/2561857/132458244-66a01811-0e5f-402f-8f61-9ba26331090f.png)

@foolwc do you have time to help fix this issue? thanks a lot

**Comment by ray-008** [comment]
> Hi @bingoku
> 
> it has already been fixed in our 2.8 version, you can use `Raw Data Editor` to configure the `gzip` plugin.
> 
> But there is another bug here, when configure `gzip` plugin in `Plugin Editor` there is an error happened:
> 
> ![image](https://user-images.githubusercontent.com/2561857/132458244-66a01811-0e5f-402f-8f61-9ba26331090f.png)
> 
> @foolwc do you have time to help fix this issue? thanks a lot

是的，这就是个bug。但是可以到路由列表，点单个路由的更多 ->查看按钮 编辑提交。

<img width="1270" alt="WX20210917-160547@2x" src="https://user-images.githubusercontent.com/4973374/133747773-2f7960d1-373c-4ab2-afeb-5e5c0878a111.png">

**Comment by bzp2010** [comment]
@ray-008 The bug can be reproduced and I will try to fix it.
